src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -55,15 +58,17 @@ export class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, S
       expanded: false,
       gridHeight: 0,
       loadFailed: false,
+      didPerformFetch: false,
     }
   }
 
   componentDidMount() {
     if (this.props.relay) {
       this.props.relay.setVariables({ fetchContent: true }, readyState => {
-        if (readyState.error) {
-          this.setState({ loadFailed: true })
-        }
+        this.setState({
+          didPerformFetch: readyState.done,
```

Comment:
Could this be false? If that would happen, does the rail ever get removed? I don't _think_ it would.